src/main/resources/xml/schema/2020-1/nordic2020-1.sch

```diff
@@ -630,6 +630,18 @@
         </rule>
     </pattern>
 
+    <pattern id="epub_nordic_253_d">
+        <title>Rule 253d</title>
+        <p>Extracted text needs to have a reference of the image the text have been extracted from.</p>
+        <rule context="html:figure[tokenize(@class, '\s+') = 'image']">
+            <let name="figdesc-id" value=".//html:aside[tokenize(@class, '\s+') = 'fig-desc']/@id" />
+            <let name="aria-id" value=".//html:img/@aria-describedby" />
+            <assert test="count(.//html:aside[tokenize(@class, '\s+') = 'fig-desc']) = 0 or $figdesc-id = $aria-id">
```

Comment:
If both attributes are missing or empty, I think this assetion will succeed? This assertion covers the second check described by Martin in #519, but the first check also needs to be implemented. That is, img elements in this context needs to have a non-empty aria-describedby attribute.

Reply:
Hi @josteinaj 

I'm not sure if I understand the finer details here, but if I look at https://format.mtm.se/nordic_epub/2020-1#text-extraction-from-images it seems like the aria-describedby is only required in the context that we have, a fig-desc (extracted text) element. And this rule will verify both that it's present and that it's equal to the description.

Is there a nuance I've missed and could you give me an example?

Best regards
Daniel